time timestamp
float64[] obstacle_x
float64[] obstacle_y
float64[] traj_x
float64[] traj_y
float64[] refPos
